.github/workflows/deploy.yml

```diff
@@ -27,27 +46,41 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
     - uses: docker/build-push-action@v4
       with:
-        tags: ${{ env.DOCKER_TAG }}:${{ env.VERSION }},${{ env.DOCKER_TAG }}:latest
+        tags: ${{ env.DOCKER_TAG }}/${{ matrix.build_target.name_suffix}}:${{ env.VERSION }},${{ env.DOCKER_TAG }}/${{ matrix.build_target.name_suffix}}:latest
         pull: true
         platforms: linux/arm64
         push: ${{ github.event_name == 'release' }}
-        file: etc/docker/Dockerfile.triton-jetpack-focal
+        file: ${{ matrix.build_target.dockerfile }}
+        build-args: ${{ matrix.build_target.build_args }}
   module:
     needs: [container]
     strategy:
       matrix:
-        platform: [arm64]
+        platform:
+          - arch: arm64
+            docker_suffix: jetpack5
+            meta_path: etc/viam/meta_jetpack5.json
+          - arch: arm64
+            docker_suffix: jetpack6
+            meta_path: etc/viam/meta_jetpack6.json
+          - arch: arm64
+            docker_suffix: cuda_arm
```

Comment:
Should the CUDA builds also name the minimum CUDA requirement? I think the version of CUDA inside the container needs to be supported by the host or it won't work. But I'm not sure. So maybe all places it says CUDA become CUDA12? Genuinely asking because I'm not sure.

Reply:
Hm... that's a good question. For a docker image name, I don't really care and am willing to add in a `12`. For the module name, my hope is that Abe's work with tags will eventually make this a non-issue, and in the meantime I can't tell what the right path forward is. So, I'm kinda inclined to leave the module name alone for now, and we can change it when the right fix becomes obvious.

I'll change the docker suffix.